src/client_waldur.py

```diff
@@ -12,13 +12,8 @@
 # {"detail":"Extension is disabled."}
```

Comment:
Do we actually need this file? I think it can be fully replaced with "ansible-waldur-module".